README.md

```diff
@@ -15,22 +15,28 @@ To accomplish this for systems hosted on cloud.gov, the code in this repository
 
 Note: Instructions currently assume you will ship to _both_ New Relic and S3. Better configuration is TODO.
 
-All of the following steps take place in the same cf space where the logshipper will reside. Commands in .profile are looking for specific service names, so use the names suggested (or edit .profile).
+All of the following steps take place in the same cf space where the logshipper will reside.
+
+Commands in .profile look for a specific tag in relation to the service. The names of the specific services can be unique, without impacting the `.profile`.
+Current Supported Tags:
+- The "newrelic-creds" user provided service = `newrelic-creds`
+- The "log-storage" s3 bucket = `logshipper-s3`
+- The "cg-logshipper-creds" user provided service = `logshipper-creds`
 
 1. Create a user-provided service "newrelic-creds" with your New Relic license key
     ```sh
-    cf create-user-provided-service newrelic-creds -p '{"NEW_RELIC_LICENSE_KEY":"[your key]", "NEW_RELIC_LOGS_ENDPOINT": "[your endpoint]"}'
+    cf create-user-provided-service my-newrelic-credential-name -p '{"NEW_RELIC_LICENSE_KEY":"[your key]", "NEW_RELIC_LOGS_ENDPOINT": "[your endpoint]"}'
```

Comment:
Please add -t to these examples to show the tags, e.g., "-t 'newrelic-creds'"

Reply:
Added to cf cli commands, good catch!